### PR TITLE
docs: add tutorial database group

### PR DESCRIPTION
**Pull request by adela-bytebase**

### CODE REVIEW

File: content/docs/tutorials/batch-change-with-database-group.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+   - `demo_db_test` in `Test` environment and it will run automatically. Because by default, issue in `Test` environment will rollout automatically;
+   - `demo_db_prod_asia` in `Prod` environment and click **Rollout**;
+   - `demo_db_prod_europe` in `Prod` environment and click **Rollout**;
+   - `demo_db_prod_africa` in `Prod` environment and click **Rollout**;
+   - `demo_db_prod_australia` in `Prod` environment and click **Rollout**;
+   - `demo_db_prod_north_america` in `Prod` environment and click **Rollout**;
+   - `demo_db_prod_south_america` in `Prod` environment and click **Rollout**;
+   - `demo_db_prod_antarctica` in `Prod` environment and click **Rollout**.
+
+## Step 3 - Group Databases and Run Batch Change
```
Comment (tianzhou) [inline content/docs/tutorials/batch-change-with-database-group.md:70]: Group Databases and Create Tables in Batch

File: content/docs/tutorials/batch-change-with-database-group.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+![bb-issue-7-db-t1-one-db-passed](/content/docs/tutorials/batch-change-with-database-group/bb-issue--7-db-t1-one-db-passed.webp)
+
+4. Repeat 2 to 3 to create another issue to create another table `t2`:
+   ```sql
+   CREATE TABLE t2 (
+      id INTEGER PRIMARY KEY
+   );
+   ```
+
+## Step 4 - Group Tables and Run Batch Change
```
Comment (tianzhou) [inline content/docs/tutorials/batch-change-with-database-group.md:100]: Group Tables and Alter tables in Batch

File: content/docs/tutorials/batch-change-with-database-group.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+4. Here we choose `demoall_t` which includes `t1` and `t2`. Copy and paste the SQL below and click **Create**. 
+   ```sql
+   ALTER TABLE demoall_t ADD COLUMN name VARCHAR(255) NOT NULL;
+   ```
+
+5. On the issue page, click the **down arrow** and choose **Rollout** to run one database first. If it's OK, then click **Rollout current stage** to run all.
+
+
+## Summary
+Now you have learned how to use database group to run batch change in Bytebase. Bytebase provides more other ways to help you run batch changes smoothly. Please refer to [Batch Change](/docs/change-database/batch-change/) for more details.
```
Comment (tianzhou) [inline content/docs/tutorials/batch-change-with-database-group.md:123]: database group and table group to run batch changes

**Review by tianzhou — CHANGES_REQUESTED** [review]
Also need to add to the tutorial overview https://github.com/bytebase/bytebase.com/blob/main/content/docs/tutorials/overview.md

File: content/docs/tutorials/batch-change-with-database-group.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+tags: Tutorial
+level: Intermediate
+description: This article briefly describes the general scenarios of database schema synchronization and how to use this feature smoothly in Bytebase with pure UI operations.
+---
+
+Bytebase [2.2.0](/changelog/bytebase-2-2-0/) introduced **database grouping**, a new feature that provides a third way to facilitate batch change, in addition to **multiple environments** and **multiple tenants**.
+
+## Step 1 - Start Bytebase and MySQL
+
+0. Make sure you installed and running [Docker](https://www.docker.com/).
```
[the remark this answers is not in the document]
Comment (adela-bytebase) [reply content/docs/tutorials/batch-change-with-database-group.md]: Yes, but the classic "Prerequisites" might be better still hahah! I change it!

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**